src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -671,6 +671,9 @@ Microsoft.CodeAnalysis.Semantics.UnaryOperationKind.UnsignedPostfixDecrement = 7
 Microsoft.CodeAnalysis.Semantics.UnaryOperationKind.UnsignedPostfixIncrement = 771 -> Microsoft.CodeAnalysis.Semantics.UnaryOperationKind
 Microsoft.CodeAnalysis.Semantics.UnaryOperationKind.UnsignedPrefixDecrement = 774 -> Microsoft.CodeAnalysis.Semantics.UnaryOperationKind
 Microsoft.CodeAnalysis.Semantics.UnaryOperationKind.UnsignedPrefixIncrement = 773 -> Microsoft.CodeAnalysis.Semantics.UnaryOperationKind
+Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveGenericsOptions(Microsoft.CodeAnalysis.SymbolDisplayGenericsOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
+Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveLocalOptions(Microsoft.CodeAnalysis.SymbolDisplayLocalOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
+Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveMiscellaneousOptions(Microsoft.CodeAnalysis.SymbolDisplayMiscellaneousOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
```

Comment:
> Are there here just for consistency?

Yes.

> If we're going to be consistent, are we going to have Remove APIs whenever we have Add APIs?

It seemed like that was already the intention since there were already a handful of `Remove*` methods. Since I was adding one more, I thought adding the last two would be the right thing to do. Taking a broader view, it seems odd to have full set of `With*` methods, a full set of `Add*` helpers (which are arguably are even simpler) but an incomplete set of `Remove*`.

> Was there a discussion on whether we need those new public APIs?
I wouldn't quite call it a discussion, but @CyrusNajmabadi took a look at API changes and agreed that these seemed like unintentional omissions.